packages/trader/src/App/Components/Elements/ContractDrawer/contract-drawer.tsx

```diff
@@ -133,10 +130,7 @@ const ContractDrawer = observer(
                 <div
                     id='dt_contract_drawer'
                     className={classNames('contract-drawer', {
-                        'contract-drawer--with-collapsible-btn':
-                            !!getEndTime(contract_info) ||
-                            ((is_multiplier || is_vanilla || is_turbos || is_accumulator || is_smarttrader_contract) &&
-                                isMobile()),
+                        'contract-drawer--with-collapsible-btn': !!getEndTime(contract_info) || is_mobile,
```

Comment:
I don't think, that replacing i`sMobile()` with `is_mobile` won't break anything in this case (so it looks good to me), but actually they have different 'breakpoints': 926px for `isMobile()` and 767 for `is_mobile`. 
Anyway, it's good that you replace it, as isMobile() is going to be deprecated